src/icons/clear.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+function ClearIcon(props) {
```

Comment:
Generally we use `const ClearIcon = (props) => ()` style even for functional components like this one.